tests/css/test_stylesheet.py

```diff
@@ -101,3 +101,47 @@ def test_did_you_mean_for_css_property_names(
             f'Did you mean "{expected_property_name_suggestion}"?'
         )
         assert help_text.bullets[0].markup == expected_suggestion_message
+
+
+@pytest.mark.parametrize(
+    "css_property_name,css_property_value,expected_color_suggestion",
+    [
+        ["color", "blu", "blue"],
+        ["background", "chartruse", "chartreuse"],
+        ["tint", "ansi_whi", "ansi_white"],
+        ["scrollbar-color", "transprnt", "transparent"],
+        ["color", "xkcd", None],
+    ],
+)
+def test_did_you_mean_for_color_names(
+    css_property_name: str, css_property_value: str, expected_color_suggestion
+):
+    stylesheet = Stylesheet()
+    css = """
```

Comment:
Could this be an f-string?

Reply:
It could! Using `.replace()` this way with placeholders is not great for sure, but using f-strings would require escaping all the curly braces in the "CSS" string, which is not great either :grimacing: